### PR TITLE
US134135: FACE Support for Imported SCORM

### PR DESCRIPTION
**Pull request by coffeexcode**
## Relevant Rally
- [US134135](https://rally1.rallydev.com/#/?detail=/userstory/618457828675&fdp=true): SCORM - Properties can be edited and saved

## Description
Basic FACE page for imported SCORM.
Contains

- Editing
  - Title, due date, availability dates
- Saving 
- Cancelling

The rest of the features have separate items to complete.

## Related PRs
1. LMS: https://github.com/Brightspace/lms/pull/17714
2. Siren: https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/445

### CODE REVIEW

**Review by ianlaidlaw — DISMISSED** [review]

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import '../shared-components/d2l-activity-content-editor-title.js';
+import './d2l-activity-content-imported-scorm-display-options.js';
+import { shared as activityStore, sharedScoring as scoringStore } from '../../state/activity-store.js';
```
Comment (ChrisKraljevicD2L) [inline components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js:3]: don't need `sharedScoring` right now, remove pls

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
+
+class ContentImportedScormActivityDetail extends AsyncContainerMixin(SkeletonMixin(ErrorHandlingMixin(LocalizeActivityEditorMixin(EntityMixinLit(RtlMixin(ActivityEditorMixin(MobxLitElement))))))) {
+
+	static get properties() {
+		return {
+			activityUsageHref: { type: String },
+			scoreEditorLoading: { type: Boolean }
```
Comment (ChrisKraljevicD2L) [inline components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js:24]: also not needed for now

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+			</label>
+			<label class="d2l-input-radio-label d2l-skeletize">
+				<input
+					id="open-new-tab"
+					type="radio"
+					name="link-display-group"
+					value="newTab"
+					?checked="${isExternalResource}"
+					@change="${this._saveLinkOptions}">
+					${this.localize('content.openNewTab')}
```
Comment (Kiefer-Co-D2L) [inline components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js:109]: Nitpick!  Is this line and the `span` below it indented too far by one level?

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+				:host([dir="rtl"]) #score-container {
+					margin-left: 40px;
+					margin-right: 0;
+				}
+				.d2l-grade-type-text {
+					margin: 0 0.75rem 0 0.2rem;
+				}
+				:host([dir="rtl"]) .d2l-grade-type-text {
+					margin: 0 0.2rem 0 0.75rem;
+				}
```
Comment (ChrisKraljevicD2L) [inline components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js:58]: rework CSS for new score and duedate layout changes in design

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js (hunk 3 above)
Comment (coffeexcode) [reply components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js:109]: Github formatting maybe?
image

Comment (Kiefer-Co-D2L) [reply components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js:109]: Yeah, looks like github formatting to me.  Your screenshot is good.

Comment (Kiefer-Co-D2L) [reply components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js:109]: The `<input>` tag doesn't have a closing tag, so I would say the `this.localize` and the span aren't inside it and don't need to be indented.

Comment (coffeexcode) [reply components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-display-options.js:109]: Probably going to leave it as is for this PR. Looks like the same for every other activity type that has external resource capability. Maybe a thing for dragon slayer next week?

**Review by Kiefer-Co-D2L — APPROVED** [review]
Fixes LGTM!

**Pull request merged**